src/common/selectors.ts

```diff
@@ -0,0 +1,246 @@
+import type { Page } from "@playwright/test";
+
+export const selectors = {
+    dashboard: {
+        parent: "dashboard",
+        elements: {
+            target: "div[role=\"main\"] >> text=Clear and preload cache"
+        }
+    },
+    cache: {
+        parent: "cache",
+        elements: {
+            mobileDeviceCache: {
+                checkbox: "#cache_mobile",
+                target: "label[for=cache_mobile]"
+            },
+            mobileDeviceSeparateCache: {
+                checkbox: "#do_caching_mobile_files",
+                target: "label[for=do_caching_mobile_files]"
+            },
+            cacheLoggedUser: {
+                checkbox: "#cache_logged_user",
+                target: "label[for=cache_logged_user]"
+            }
+        }
+    },
+    fileOptimization: {
+        parent: "file_optimization",
+        elements: {
+            minifyCss: {
+                checkbox: "#minify_css",
+                target: "label[for=minify_css]",
+                after: async (page: Page, state: boolean): Promise<void> => {
```

Comment:
@jeawhanlee I think we miss understood for the the side case where we can use `after` filed was only RUCSS.

For the other we can create a custom filed type that we handle.

If it is not possible we could handle it using an high order function: 
```js
function clickPopUp(string $text) {
   return async (page: Page, state: boolean): Promise<void> => {
                    if (!state) {
                        return;
                    }

                    await page.waitForSelector(`text=${text}`);
                    await page.locator(`text=${text}`).click();
                };
}
```

This would prevent repeating the code multiple times. :D

Reply:
@CrochetFeve0251 I noticed the WET code there too 😁. A high order function would be better.